backend/homework/8/demo/src/main/java/shrishti/example/demo/Repository/Inventory.java

```diff
@@ -0,0 +1,49 @@
+package shrishti.example.demo.Repository;
+
+import lombok.Data;
+
+import shrishti.example.demo.Entity.Vehicle;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class Inventory {
+private List<Vehicle>vehicles  = new ArrayList<>();
+public void save(Vehicle vehicle){
+    this.vehicles.add(vehicle);
+}
+public Vehicle findById(int id){
+    return vehicles.get(id);
+}
+
+public void updateVehicle(int id, Vehicle vehicle){
+    vehicles.get(id).setBrandName(vehicle.getBrandName());
```

Comment:
Is it possible that the object will be null? Consider a handling mechanism for that scenario.